common-dup/src/main/java/org/duracloud/mill/dup/DuplicationPolicy.java

```diff
@@ -46,7 +62,11 @@
     }
 
     public Set<DuplicationStorePolicy> getDuplicationStorePolicies(String spaceId) {
-        return spaceDuplicationStorePolicies.get(spaceId);
+        if(!spacesToIgnore.contains(spaceId)){
```

Comment:
It doesn't look like `spacesToIgnore` will ever be anything but an empty list. There is not a setter method, and none of the changes in this method appear to add anything to that list.

Reply:
The marshalling is handled by a Jackson ObjectMapper.  There are unit tests that prove that marshalling and unmarshalling of spacesToIgnore works.